DQM/L1TMonitor/src/L1TStage2EMTF.cc

```diff
@@ -742,7 +746,7 @@ void L1TStage2EMTF::analyze(const edm::Event& e, const edm::EventSetup& c) {
         int ring = TrkHit.Ring();
         int station = TrkHit.Station();
         int sector = TrkHit.Sector();
-        int subsector = TrkHit.Subsector();
+        //int subsector = TrkHit.Subsector();
```

Comment:
Do you have a good reason to keep this commented code? If not, please delete the comment lines.